### PR TITLE
PST-017-add-notifications

### PR DESCRIPTION
**Pull request by skrzypczyk-pawel**
The most important files in this ticket.

### CODE REVIEW

File: src/components/Todo/Todo.tsx [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+import React, { FC, useState } from "react";
 
 import { ITodo, IconName } from "src/types";
 import { StyleSheet, css } from "aphrodite";
 import { colors, getShadow, typography } from "src/styles";
 import { i18n } from "src/locale";
 import { Icon } from "src/components/Icon/Icon";
+import { useDispatch } from "react-redux";
+import { changeStatus, deleteTodo } from "src/store/todo/actions";
+import { useNotification } from "src/hooks/useNotification";
```
Comment (Stared55) [inline src/components/Todo/Todo.tsx]: index

File: src/components/Todo/TodoList.tsx [hunk 2]
```diff
@@ -3,28 +3,34 @@ import { useDispatch, useSelector } from "react-redux";
 import { Todo } from "src/components/Todo/Todo";
 import Loader from "src/components/Loader";
 import { fetchTodoRequest } from "src/store/todo/actions";
-import { i18n } from "src/locale";
 import { AppState } from "src/store/rootReducer";
 import { ITodo } from "src/types";
+import { useNotification } from "src/hooks/useNotification";
```
Comment (Stared55) [inline src/components/Todo/TodoList.tsx]: index

File: src/store/todo/reducer.tsx [hunk 3]
```diff
@@ -34,7 +37,22 @@ export default (state = initialState, action: TodoActions) => {
...
+        todos: [...state.todos, action.payload],
+      };
+    case DELETE_TODO:
+      return {
+        ...state,
+        todos: [...state.todos].filter((task) => task.id !== action.payload),
+      };
+    case CHANGE_STATUS:
+      const updatedTodos = [...state.todos];
+      const index = updatedTodos.findIndex(
```
Comment (Stared55) [inline src/store/todo/reducer.tsx]: I think u can use find isntead of findIndex and then remove 54th line

**Review by Stared55 — APPROVED** [review]
fix and ✅

**Review by MRadek55 — APPROVED** [review]

**Pull request merged**